tests/test_flows.py

```diff
@@ -261,6 +261,37 @@ def test_ap_config_renew(self):
         if not bssid1:
             self.fail("repeater1 didn't configure Multi-AP-24G-1")
 
+    def test_ap_config_bss_tear_down(self):
+        # Configure the controller and send renew
+        self.gateway_ucc.cmd_reply("DEV_RESET_DEFAULT")
+        self.gateway_ucc.cmd_reply(
+            "DEV_SET_CONFIG,bss_info1,"
+                "{self.mac_repeater1} 8x Multi-AP-24G-3 0x0020 0x0008 maprocks1 0 1".format(self = self))
+        self.gateway_ucc.dev_send_1905(self.mac_repeater1, 0x000A,
+            tlv(0x01, 0x0006, "{" + self.mac_gateway + "}"),
+            tlv(0x0F, 0x0001, "{0x00}"),
+            tlv(0x10, 0x0001, "{0x00}"))
+
+        # Wait a bit for the renew to complete
+        time.sleep(3)
+
+        self.check_log(self.repeater1, "agent_wlan0", r"Received credentials for ssid: Multi-AP-24G-3 .* bss_type: 2")
+        self.check_log(self.repeater1, "agent_wlan2", r".* tear down radio")
+        # TODO check that conn map was updated on controller
+
+        # SSIDs have been removed for the CTT Agent1's front radio
+        self.gateway_ucc.cmd_reply(
+            "DEV_SET_CONFIG,bss_info1,{self.mac_repeater1} 8x".format(self = self))
+        # Send renew message
+        self.gateway_ucc.dev_send_1905(self.mac_repeater1, 0x000A,
+            tlv(0x01, 0x0006, "{" + self.mac_gateway + "}"),
+            tlv(0x0F, 0x0001, "{0x00}"),
+            tlv(0x10, 0x0001, "{0x00}"))
+
+        time.sleep(3)
+        self.check_log(self.repeater1, "agent_wlan0", r".* tear down radio")
+        # TODO check that conn map was updated on controller
```

Comment:
There should be no need to trigger topology query, since the controller should do that automatically upon receiving a topology notification -> #925.

I added the TODO because I removed the corresponding code that existed in test_flows.sh, and then reimplemented it in a different way in the next commit. Since it wasn't a one-to-one mapping I thought it was better to do it in a separate commit, and I didn't want to bother with implementing it the original way first.